worth2/templates/base.html

```diff
@@ -55,27 +58,31 @@
     </ul>
 
     <ul class="nav navbar-nav navbar-right">
-        {% if request.user.profile.is_participant %}
+        {% if user.profile.is_participant %}
```

Comment:
I see that the "anonymous" check is further down. Does this break if the user is anonymous?